bin/run.sh

```diff
@@ -39,7 +39,13 @@ test_output=$(bash ${solution_dir}/test.sh ${slug} 2>&1)
 # Write the results.json file based on the exit code of the command that was 
 # just executed that tested the implementation file
 if [ $? -eq 0 ]; then
```

Comment:
If you're using back, you can use `if (( $? == 0 ))` ... but that can also be another PR.